sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/DateExpressionsSuite.scala

```diff
@@ -79,30 +79,22 @@ class DateExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
   }
 
   test("datetime function current_date") {
```

Comment:
This test and below are supposed to be positive tests. Do we have positive tests for the functions `current_date()`, `current_timestamp()`, `localtimestamp()` in Spark code base?

Reply:
Actually these tests were not useful at all previously, as they are testing dead code. We should check `ComputeCurrentTimeSuite` and make sure it has good test coverage for the execution part.